CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ of response and not the raw API endpoint response
       - `@cumulus/es-client/indexer.upsertExecution` to upsert an execution
       - `@cumulus/es-client/indexer.upsertPdr` to upsert a PDR
       - `@cumulus/es-client/indexer.upsertGranule` to upsert a granule
+  - **CUMULUS-2510**
+    - Added `collection_sns_topic_arn` environment variable to
+      `api.tf`.
```

Comment:
```suggestion
    - Added `collection_sns_topic_arn` environment variable to
      `PrivateApiLambda` and `ApiEndpoints` lambdas
```

The more important thing to note is what user facing things were changed, not what files were edited necessarily